src/impl/tpls/KokkosBlas_Cuda_tpl.hpp

```diff
@@ -0,0 +1,24 @@
+#include<KokkosBlas_tpl_spec.hpp>
+
+namespace KokkosBlas {
+namespace Impl {
+
+CudaBlasSingleton::CudaBlasSingleton()
+{    
+    cublasCreate( & handle );
+    cublasStatus_t stat = cublasCreate(&handle);
+    if (stat != CUBLAS_STATUS_SUCCESS)
+        Kokkos::abort("CUBLAS initialization failed\n");
+    printf("cuBLAS context initialized.\n");
+}
+
+CudaBlasSingleton::~CudaBlasSingleton()
+{
+    cublasDestroy(handle); printf("cuBLAS context released.\n");
+}
+
+CudaBlasSingleton & CudaBlasSingleton::singleton()
+{ static CudaBlasSingleton s ; return s ; }
```

Comment:
Do we need to do this via a pointer or a create function or something. cublasCreate needs to be called after Kokkos::initialize().

Reply:
@crtrott cublasCreate is already called after Kokkos::initialize()